.github/scripts/requirements.txt

```diff
@@ -14,4 +14,4 @@
 # limitations under the License.
 PyYAML~=6.0
 pytz==2024.2
-requests==2.32.3
+requests==2.32.4
```

Comment:
@yunchipang Could you please add comments to `tests/setup.py` and this file to remind developers that they should be updated consistently.